smaug/core/tensor.h

```diff
@@ -414,38 +511,59 @@ class Tensor : public TensorBase {
         }
     }
 
-    // Return a TensorProto that serializes this Tensor.
+    /** Serializes this Tensor to a TensorProto. */
     TensorProto* asTensorProto();
 
+    /**
+     * Returns a const pointer to the Tensor data.
+     */
+    // TODO(xyzsam): Should this be const T*?
```

Comment:
Yeah, I think here we want to return a pointer to const T data. So this should be `const T*`.

Reply:
I'll put all code-related fixes into a separate commit so that this commit only deals with docs.